pyproject.toml

```diff
@@ -23,7 +23,7 @@ include = ["openapi_pydantic/py.typed"]
 
 [tool.poetry.dependencies]
 python = "^3.8"
-pydantic = "^1.8.2"
+pydantic = "*"
```

Comment:
Lets keep the version a little more constrained - we don't want to support ancient versions (1.8 is 2 years old and removed a bunch of legacy bits).

```suggestion
pydantic = ">=1.8"
```

Reply:
Agreed and done.